packages/render/src/types.ts

```diff
@@ -110,21 +109,24 @@ export type FrameState = {
   isLoading?: null | FrameStackPending;
   inputText: string;
   setInputText: (s: string) => void;
-  onButtonPress: (frameButton: FrameButton, index: number) => void;
+  onButtonPress: (
+    frameButton: FrameButton,
+    index: number
+  ) => void | Promise<void>;
   /** Whether the frame at the top of the stack has any frame validation errors. Undefined when the frame is not loaded or set */
   isFrameValid: boolean | undefined;
   frameValidationErrors: Record<string, string[]> | undefined | null;
-  error: null | unknown;
+  error: unknown;
   homeframeUrl: string | null;
 };
 
-export type onMintArgs = {
+export type OnMintArgs = {
   target: string;
   frameButton: FrameButton;
   frame: Frame;
 };
 
-export type onTransactionArgs = {
+export type OnTransactionArgs = {
```

Comment:
these are breaking changes no?

Reply:
Yes you are right, regarding the types it is a breaking change.

Reply:
Will add a changeset to do a minor